### PR TITLE
feat(style-guide): Resource box styling

### PR DESCRIPTION
**Pull request by neha-ot**

### CODE REVIEW

File: style-guide/src/components/handover-process/ResourceBox.js [hunk 1]
```diff
@@ -4,12 +4,12 @@ import copy from '../../../static/handover-process';
 import styles from '../../styles/handover.module.scss';
 
 export default ({ resource }) => (
-  <div className={styles.resource}>
+  <div className={styles.resource} onClick={() => { window.open(copy[`${resource}.link`], '_blank')}}>
```
Comment (ronniechong) [inline style-guide/src/components/handover-process/ResourceBox.js:7]: Semantically, `<a>` is more suitable for this as it is an actionable element

Comment (jakeclements) [inline style-guide/src/components/handover-process/ResourceBox.js:7]: Can this be either an anchor tag or a button? It'd be more semantically correct than having a click handler on a presentational element

**Review by jakeclements — APPROVED** [review]
Thanks Neha

**Review by ronniechong — APPROVED** [review]
There!

**Pull request merged**